src/bptl/work_units/brp/tasks.py

```diff
@@ -0,0 +1,39 @@
+from bptl.tasks.base import WorkUnit
+from bptl.tasks.registry import register
+
+from .client import get_client_class
+
+__all__ = ["IsAboveAge"]
+
+
+@register
+class IsAboveAge(WorkUnit):
+    """
+    Fetches BRP API and return flag if a certain person is older above a certain age
```

Comment:
This help text gets picked up by the UI right? In that case, I think we should also mention here it's "equal or above".

```suggestion
    Fetches BRP API and returns whether a person is exactly, or older than, a certain age.
```